BooruSharp/Booru/Template/Sankaku.cs

```diff
@@ -63,6 +81,18 @@ private protected override Search.Post.SearchResult GetPostSearchResult(JToken e
                 );
         }
 
+        private Search.Tag.TagType GetTagType(int type)
+        {
+            switch(type)
+            {
+                case 1: return Search.Tag.TagType.Artist;
+                case 3: return Search.Tag.TagType.Copyright;
+                case 4: return Search.Tag.TagType.Character;
+                case 8: return Search.Tag.TagType.Metadata;
+                default: return Search.Tag.TagType.Trivia;
```

Comment:
I dno't know how the API behave but I think it's better to do a case for Trivia and throw an error in default, so if a new tag is created we will know it

Reply:
There are already new tag categories in many boorus but I don't see the value in finding them all when trivia will just do.

Reply:
You only changed tag parsing for Sankaku and E621 or did I miss smth?

Reply:
I changed it for all that had those tag categories

Reply:
All in all I think it's better like this, adding each individual tag category is just a lot of hassle for very little gain. I wanna change gelbooru to do the same too.

Reply:
Ah I didn't notice E621 change their tag system, Gelbooru still use the "old" one right, what is there to change?

Reply:
Uhm... Overall I think adding the others tags could be a neat feature, if it's just that it's a hassle to you, you can let it like that and I'll change this part later myself

Reply:
Cool. Gelbooru at the moment will not list any new tag categories that might pop up, instead they are just not even noticed in any way.

Reply:
Oh sorry it was danbooru I was talking about, gelbooru has no tags at all in the response